manual/manual/refman/expr.etex

```diff
@@ -430,6 +430,14 @@ and allow one to use this polymorphism in recursive occurrences
 (when using @"let" "rec"@). Note however that this is a normal polymorphic
 type, unifiable with any instance of itself.
 
+It is possible to define local exceptions in expressions:
+@ "let" exception constr-decl "in" expr @ .
+The syntactic scope of the exception constructor is the inner
+expression, but nothing prevents exception values created with this
+constructor from escaping this scope.  Two executions of the definition
+above result in two incompatible exception constructors (as for any
+exception definition).
```

Comment:
I'm not clear on what this last sentence means.

Reply:
An example might help:
```OCaml 
let f () = let exception A in A
;; assert ( f () <> f ())
```

Reply:
A good example might assist in context, yes.